src/screens/ErrorScreen/styles.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../scss/variables";
+@import "../../scss/_base";
+
+#error-screen-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.big-text {
```

Comment:
I think its good to nest styles in scss.
like
#error-screen-div {
  .....
  .big-texx {
      .....
   }
}

Reply:
Oh I was not familiar with how scss worked so i used the default css syntax